packages/xo-web/src/common/xo/index.js

```diff
@@ -301,10 +301,11 @@ export const subscribeCurrentUser = createSubscription(() => xo.refreshUser())
 
 export const subscribeAcls = createSubscription(() => _call('acl.get'))
 
-export const subscribeHvSupportedVersions =
-  xoaPlans.CURRENT !== xoaPlans.SOURCES && xoaPlans.CURRENT !== xoaPlans.UNKNOWN
-    ? createSubscription(() => _call('xoa.getHVSupportedVersions'))
-    : noop
+export const subscribeHvSupportedVersions = createSubscription(() => {
+  if (xoaPlans.CURRENT !== xoaPlans.SOURCES && xoaPlans.CURRENT !== xoaPlans.UNKNOWN) {
+    return _call('xoa.getHVSupportedVersions')
+  }
+})
```

Comment:
Not Perfect, but at least it's a valid subscription :+1:

You could move the test out of the callback though.

Reply:
If am moving the test out of the callback, it trigger an error in the web console in case of the test don't past. (cb is not a fonction)

(Eg:
```typescript
createSubscription(
  xoaPlans.CURRENT !== xoaPlans.SOURCES &&
    xoaPlans.CURRENT !== xoaPlans.UNKNOWN &&
    (() => _call('xoa.getHVSupportedVersions'))
)
```
)

Reply:
It's because you cannot pass a boolean to `createSubscription`, you can do something like:

```suggestion
export const subscribeHvSupportedVersions = createSubscription(
  xoaPlans.CURRENT !== xoaPlans.SOURCES &&
    xoaPlans.CURRENT !== xoaPlans.UNKNOWN
    ? () => _call('xoa.getHVSupportedVersions')
    : noop
)
```

Reply:
But what is the gain to move the test outside ?
Because in this case, the "final result" is exactly the same no ?

Reply:
Yes, it will be, but it will avoid running the test every 5 seconds.